### PR TITLE
New language! Spanish

### PR DESCRIPTION
**Pull request by carlesvimas**
Language added following this guide [.](https://github.com/Lattyware/massivedecks/wiki/Translation). I'm pretty sure it's ready for implementation

### CODE REVIEW

**Comment by carlesvimas** [comment]
Would love to see it live since I really enjoy this game despite the lack of my language.

**Comment by Lattyware** [comment]
Thanks for this contribution! Looks great.
I've been ill for a bit and so not sure when I'll get the moment to merge this through, but I'll do so as soon as possible.

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 1]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Saber más" ]
+
+        AboutTheGameDescription ->
+            [ Text "Saber más sobre ", ref MassiveDecks, Text " ." ]
+
+        MDLogoDescription ->
+            [ Text "Una ", ref (noun Call 1), Text " y una ", ref (noun Response 1), Text " marcadas con una "M" y con una "D"." ]
+
+        RereadLogoDescription ->
+            [ Text "Un libro enecerrado en un circulo por una flecha de reciclaje." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:132]:
```suggestion
            [ Text "Un libro encerrado en un circulo por una flecha de reciclaje." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 2]
```diff
@@ -0,0 +1,1428 @@
...
+
+        MDProject ->
+            [ Text "El proyecto de github" ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:135]:
```suggestion
            [ Text "El proyecto de Github" ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 3]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Desarrollado por ", ref RereadGames, Text "." ]
+
+        RereadGames ->
+            [ Text "Reread Games" ]
+
+        NameLabel ->
+            [ Text "Tu Nombre" ]
+
+        NameInUse ->
+            [ Text "Otra persona esta usando este nombre en la partida—porfavor elija otro diferente." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:150]:
```suggestion
            [ Text "Otra persona está usando este nombre en la partida—por favor elija otro diferente." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 4]
```diff
@@ -0,0 +1,1428 @@
...
+
+        RejoinTitle ->
+            [ Text "Volver a unirse a una partida" ]
+
+        RejoinGame { code } ->
+            [ Text "Volver a unirse a “", GameCode { code = code } |> ref, Text "”." ]
+
+        LobbyRequiresPassword ->
+            [ Text "Necesitas una contraseña para entrar a la partida. Preguntasela a la persona que te ha invitado." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:159]:
```suggestion
            [ Text "Necesitas una contraseña para entrar a la partida. Pídesela a la persona que te ha invitado." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 5]
```diff
@@ -0,0 +1,1428 @@
...
+
+        Copy ->
+            [ Text "Copiar" ]
+
+        -- Rules
+        CardsAgainstHumanity ->
+            [ Text "Cartas Contra La Humanidad" ]
+
+        Rules ->
+            [ Text "Como jugar." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:175]:
```suggestion
            [ Text "Cómo jugar." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 6]
```diff
@@ -0,0 +1,1428 @@
...
+            , Text " "
+            , ref (noun Point cost)
+            , Text " para descartar tu baraja y obtener una nueva."
+            ]
+
+        HouseRuleRebootCost ->
+            [ ref (noun Point 1), Text " Coste" ]
+
+        HouseRuleRebootCostDescription ->
+            [ Text "Cuantas ", ref (nounUnknownQuantity Point), Text " cuesta reiniciar la baraja." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:271]:
```suggestion
            [ Text "Cuántos ", ref (nounUnknownQuantity Point), Text " cuesta reiniciar la baraja." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 7]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Añade "
+            , ref (nounUnknownQuantity Response)
+            , Text " en blanco, donde los jugadores podran diseñar cartas blancas nuevas."
+            ]
+
+        HouseRuleComedyWriterNumber ->
+            [ Text "Blank ", ref (nounUnknownQuantity Response) ]
+
+        HouseRuleComedyWriterNumberDescription ->
+            [ Text "El numero de "
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:299]:
```suggestion
            [ Text "El número de "
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 8]
```diff
@@ -0,0 +1,1428 @@
...
+            , ref (noun Response 1)
+            , Text " en la baraja será usado como respuesta. Esta jugada pertenecerá a un bot llamado "
+            , Text "Rando Cardrissian, y si gana la partida, todos los jugadores deberán irse a casa en un estado de depressión severa."
+            ]
+
+        HouseRuleRandoCardrissianNumber ->
+            [ Text "Jugadores IA" ]
+
+        HouseRuleRandoCardrissianNumberDescription ->
+            [ Text "Numero de Bots en la partida." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:327]:
```suggestion
            [ Text "Número de Bots en la partida." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 9]
```diff
@@ -0,0 +1,1428 @@
...
+        HouseRuleHappyEndingDescription ->
+            [ Text "Cuando el juego acaba, la ultima ronda es 'Hacer un Haiku' ", ref (noun Call 1), Text "." ]
+
+        HouseRuleCzarChoices ->
+            [ ref Czar, Text " Elecciones ]
+
+        HouseRuleCzarChoicesDescription ->
+            [ Text "Al empezar la ronda, el "
+            , ref Czar
+            , Text " roba multiples "
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:349]:
```suggestion
            , Text " roba múltiples "
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 10]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Te avisará cuando tengas que hacer algo en el juego usando las notificaciones del navegador."
+            ]
+
+        NotificationsUnsupportedExplanation ->
+            [ Text "Tu navegador no soporta notificaciones." ]
+
+        NotificationsBrowserPermissions ->
+            [ Text "Tendrás que dar permiso a "
+            , ref MassiveDecks
+            , Text " para que te notifique. Esto solo se usará mientras tengas el juego abierto y esto este activado."
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:436]:
```suggestion
            , Text " para que te notifique. Esto solo se usará mientras tengas el juego abierto y mientras esto esté activado."
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 11]
```diff
@@ -0,0 +1,1428 @@
...
+            ]
+
+        NotificationOnlyWhenHiddenSetting ->
+            [ Text "Solo en segundo plano" ]
+
+        NotificationsOnlyWhenHiddenExplanation ->
+            [ Text "Solo mandará notificaciones cuando el juego esté en segundo plano(e.g: en otra página o minimizado)." ]
+
+        NotificationsOnlyWhenHiddenUnsupportedExplanation ->
+            [ Text "Tu navegador no puede comprobar la visivilidad de la página." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:446]:
```suggestion
            [ Text "Tu navegador no puede comprobar la visibilidad de la página." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 12]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Un codigo que permite a otros jugadores encontrar y unirse a la partida." ]
+
+        GameCode { code } ->
+            [ Text code ]
+
+        GameCodeSpecificDescription ->
+            [ Text "Dale este codigo a gente para que se pueda unir a la partida.." ]
+
+        GameCodeHowToAcquire ->
+            [ Text "Preguntale a la persona que te ha invitado por el ", ref GameCodeTerm, Text " de la partida." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:477]:
```suggestion
            [ Text "Pregúntale a la persona que te ha invitado por el ", ref GameCodeTerm, Text " de la partida." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 13]
```diff
@@ -0,0 +1,1428 @@
...
+
+        SettingUpGame ->
+            [ Text "Partidas sin empezar." ]
+
+        StartYourOwn ->
+            [ Text "¿Crear una nueva partida?" ]
+
+        -- Spectation
+        JoinTheGame ->
+            [ Text "¡Unirse a la sala!" ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:524]:
```suggestion
            [ Text "¡Unirse a la partida!" ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 14]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "El número de "
+            , ref (nounUnknownQuantity Point)
+            , Text " que tiene el jugador."
+            ]
+
+        Likes { total } ->
+            [ Text (String.fromInt total) ]
+
+        LikesDescription ->
+            [ Text "Numero de likes recibidos."
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:657]:
```suggestion
            [ Text "Número de \"me gusta\" recibidos."
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 15]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "las barajas deben cargarse antes de empezar el juego." ]
+
+        MissingCardType { cardType } ->
+            [ Text "Ninguna de tus barajas contiene "
+            , ref (nounUnknownQuantity cardType)
+            , Text ". Necesitas una baraja que lo tenga para empezar la partida."
+            ]
+
+        NotEnoughCardsOfType { cardType, needed, have } ->
+            [ Text "Con el número de jugadores en la partida, necesitarás almenos "
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:774]:
```suggestion
            [ Text "Con el número de jugadores en la partida, necesitarás al menos "
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 16]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Añade una baraja." ]
+
+        RemoveDeck ->
+            [ Text "Elimina una baraja." ]
+
+        SourceNotFound { source } ->
+            [ ref source, Text " no reconoce la baraja elegida. Revisa que los datos introducidos sean correctos." ]
+
+        SourceServiceFailure { source } ->
+            [ ref source, Text " ha fallado en conseguir la baraja. Porfavor prueba más tarde o cambia de proveedor." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:800]:
```suggestion
            [ ref source, Text " ha fallado en conseguir la baraja. Por favor prueba más tarde o cambia de proveedor." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 17]
```diff
@@ -0,0 +1,1428 @@
...
+
+        ManyDecks ->
+            [ Text "Many Decks" ]
+
+        ManyDecksDeckCodeTitle ->
+            [ Text "Codigo de la Baraja" ]
+
+        ManyDecksDeckCodeShort ->
+            [ Text "Un codigo de baraja tiene que ser de al menos 5 caracteres." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:809]:
```suggestion
        ManyDecksDeckCodeTitle ->
            [ Text "Código de la Baraja" ]

        ManyDecksDeckCodeShort ->
            [ Text "Un código de baraja tiene que ser de al menos 5 caracteres." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 18]
```diff
@@ -0,0 +1,1428 @@
...
+
+        HandSizeDescription ->
+            [ Text "El número base de cartas que tiene cada jugador durante la partida." ]
+
+        ScoreLimit ->
+            [ ref (noun Point 1), Text " Limit" ]
+
+        ScoreLimitDescription ->
+            [ Segment
+                [ Text "El numero de "
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:855]:
```suggestion
                [ Text "El número de "
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 19]
```diff
@@ -0,0 +1,1428 @@
...
+            ]
+
+        RandoCantWrite ->
+            [ Text "Los bots no pueden escribir sus propias cartas." ]
+
+        DisableComedyWriter ->
+            [ Text "Desabilitar ", ref HouseRuleComedyWriter ]
+
+        DisableRando ->
+            [ Text "Desabilitar ", ref HouseRuleRandoCardrissian ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:894]:
```suggestion
            [ Text "Deshabilitar ", ref HouseRuleComedyWriter ]

        DisableRando ->
            [ Text "Deshabilitar ", ref HouseRuleRandoCardrissian ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 20]
```diff
@@ -0,0 +1,1428 @@
...
+        AudienceModeDescription ->
+            [ Text "Si se activa, todos los jugadores entrarán como espectadores, y tu escogerás si cambiarlos a jugadores."
+            
+            ]
+
+        StartGame ->
+            [ Text "Empezar la partida" ]
+
+        Public ->
+            [ Text "Juego público" ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:930]:
```suggestion
            [ Text "Partida pública" ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 21]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Si se activa, el juego saldrá en el apartado de salas publicas en el menú." ]
+
+        ApplyConfiguration ->
+            [ Text "Guardar cambios." ]
+
+        AppliedConfiguration ->
+            [ Text "Guardado." ]
+
+        InvalidConfiguration ->
+            [ Text "El valor de la configuración no es valido." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:942]:
```suggestion
            [ Text "El valor de la configuración no es válido." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 22]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Cuanto tiempo (en segundos) se tiene que esperar después de que la ultima respuesta sea revelada para que empieza la siguiente fase." ]
+
+        JudgingTimeLimitDescription ->
+            [ Text "Cuanto tiempo (en segundos) el ", ref Czar, Text " tiene para escojer la mejor respuesta." ]
+
+        CompleteTimeLimitDescription ->
+            [ Text "Cuanto tiempo (en segundos) hay que esperar después de una ronda para que empieze la siguiente." ]
+
+        RevealingEnabledTitle ->
+            [ Text "Lider muestra las respuestas" ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:984]:
```suggestion
            [ Text "Zar revela las jugadas" ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 23]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "El servidor no respondió por demasiado tiempo. Puede estar caido, prueba de nuevo en un tiempo." ]
+
+        NetworkError ->
+            [ Text "Se te fué la conexión a internet, prueba más tarde." ]
+
+        ServerDownError ->
+            [ Text "El servidor está offline, porfavor prueba más tarde." ]
+
+        BadStatusError ->
+            [ Text "El servidor dio una respuesta que no esperavamos." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:1239]:
```suggestion
            [ Text "El servidor dio una respuesta que no esperábamos." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 24]
```diff
@@ -0,0 +1,1428 @@
...
+
+        BadPayloadError ->
+            [ Text "El servidor dio una respuesta que no entendimos." ]
+
+        PatchError ->
+            [ Text "El servidor nos dió una actualización que no pudimos aplicar." ]
+
+        VersionMismatch ->
+            [ Text "El servidor nos dió un cambio de configuración para una versión diferente de la que esperavamos." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:1248]:
```suggestion
            [ Text "El servidor nos dió un cambio de configuración para una versión diferente de la que esperábamos." ]
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 25]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Te has ido de la partida." ]
+
+        LobbyNotFoundError ->
+            [ Text "La partida no existe." ]
+
+        LobbyClosedError { gameCode } ->
+            [ Text "La partida a la que deseas acceder (", ref (GameCode { code = gameCode }), Text ") ha acabado." ]
+
+        LobbyDoesNotExistError { gameCode } ->
+            [ Text "El codigo que has introducido ("
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:1299]:
```suggestion
            [ Text "El código que has introducido ("
```

File: client/src/elm/MassiveDecks/Strings/Languages/Es.elm [hunk 26]
```diff
@@ -0,0 +1,1428 @@
...
+            [ Text "Alguien está usando su nombre“"
+            , Text username
+            , Text "”—prueba otro diferente"
+            ]
+
+        GameError ->
+            [ Text "Algo ha ido mal en la partida." ]
+
+        OutOfCardsError ->
+            [ Text "No havian suficientes cartas en la baraja para todos los jugadores. Prueba de añadir más en la configuración." ]
```
Comment (Polsaker) [inline client/src/elm/MassiveDecks/Strings/Languages/Es.elm:1318]:
```suggestion
            [ Text "No habían suficientes cartas en la baraja para todos los jugadores. Prueba de añadir más en la configuración." ]
```

**Review by Polsaker — CHANGES_REQUESTED** [review]
This translation probably needs some more proofreading. These were the errors I found after a quick look

**Comment by Lattyware** [comment]
This has been merged in as 651f3d1d91d0038bc9b837158de1d98182c5199b. If there are further updates or fixes, please feel free to open a new PR.

Thanks for the contribution, and sorry for the long time getting it merged in.

**Pull request closed without merging**